GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -347,6 +343,7 @@ gdjs.WebsocketDebuggerClient.prototype._circularSafeStringify = function(
 ) {
   return JSON.stringify(
     obj,
+      // @ts-ignore
```

Comment:
That one is weir as it says the arguments are from the wrong type, but the types correspond. To be more precise it says that basically `DebuggerClientCycleReplacer !== DebuggerClientCycleReplacer` wich isn't true.

Reply:
Weird, no big deal then.